Responsive Updates and Slight Aesthetic improvements

limeken opened the pull request:
This update changes both the calendar and add assignment/import buttons to have responsive versions that switch in/out when breakpoints are reached. More specifically tailwind's lg breakpoint.
The calendar switches to a vertical mode.
The "add assignment" and "import buttons" switch location and size.

On mobile versions, this will make the application more functional and user-friendly.

limeken committed:
Responsive Updates and Slight Aesthetic improvements

This update changes both the calendar and add assignment/import buttons to have responsive versions that switch in/out when breakpoints are reached.
More specifically tailwind's lg breakpoint.
The calendar switches to a vertical mode.
The "add assignment" and "import buttons" switch location and size.

On mobile versions, this will make the application more functional and user-friendly.

limeken committed:
Responsive Updates and Slight Aesthetic improvements

This update changes both the calendar and add assignment/import buttons to have responsive versions that switch in/out when breakpoints are reached.
More specifically tailwind's lg breakpoint.
The calendar switches to a vertical mode.
The "add assignment" and "import buttons" switch location and size.

On mobile versions, this will make the application more functional and user-friendly.

limeken committed:
Responsive changes to title and instructions button

Gave the instructions button ("?") some left padding so that the responsive mobile version doesn't press the question mark icon against the left side of the page.

Also changed the title such that the text size scales with the size of the window. Allowing for a more responsive and useable website.

limeken committed:
Responsive changes to title and instructions button

Gave the instructions button ("?") some left padding so that the responsive mobile version doesn't press the question mark icon against the left side of the page.

Also changed the title such that the text size scales with the size of the window. Allowing for a more responsive and useable website.

superglubglub committed:
Merge remote-tracking branch 'origin/responsive' into responsive

superglubglub committed:
serialisation of assessments

superglubglub committed:
Merge remote-tracking branch 'origin/main' into responsive

# Conflicts:
#	AssignmentCalculator/package-lock.json
#	AssignmentCalculator/src/App.tsx
#	AssignmentCalculator/src/components/NotificationBanner.tsx
#	AssignmentCalculator/src/components/Submission.tsx
#	AssignmentCalculator/src/components/UniversityBanner.tsx
#	AssignmentCalculator/src/components/calendar/Calendar.tsx
#	AssignmentCalculator/src/components/calendar/CalendarOptions.tsx
#	AssignmentCalculator/src/components/calendar/TextCalendar.tsx
#	AssignmentCalculator/src/components/calendar/VisualCalendar.tsx
#	AssignmentCalculator/src/index.css

superglubglub committed:
fixed some styling stuff and merged with main for coolneess

superglubglub committed:
Merge remote-tracking branch 'origin/main' into responsive

# Conflicts:
#	AssignmentCalculator/src/App.tsx
#	AssignmentCalculator/src/components/Submission.tsx
#	AssignmentCalculator/src/components/calendar/Calendar.tsx
#	AssignmentCalculator/src/components/calendar/CalendarOptions.tsx
#	AssignmentCalculator/src/components/calendar/PriorityQueue.tsx
#	AssignmentCalculator/src/components/calendar/VisualCalendar.tsx

superglubglub committed:
merged with main for pr

superglubglub committed:
Merge remote-tracking branch 'origin/main' into responsive

# Conflicts:
#	AssignmentCalculator/src/components/Instructions.tsx
#	AssignmentCalculator/src/components/Submission.tsx
#	AssignmentCalculator/src/components/calendar/VisualCalendar.tsx

MagMicro commented:
The styling file is nowhere to be found causing crashes and preventing vercel. Is it a file that didnt get properly commited first?

superglubglub committed:
changed uwa brand color and added styling file

superglubglub committed:
changed uwa brand color and added styling file

superglubglub committed:
fixed some mobile stuff

MagMicro committed:
various visual changes

limeken committed:
Updating Hover and Click Aesthetic.

This simple change should make the "day nodes" feel more clickable to the user, thereby indicating and giving a more intuitive visual design.

Selected days are now highlighted too. With the uwaBlue colour.

Other crucial visual updates too. Mostly to buttons and colours,

superglubglub committed:
Merge branch 'main' into responsive

# Conflicts:
#	AssignmentCalculator/src/index.css
